startup.cmd

```diff
@@ -9,7 +9,7 @@ SET SAVE_PYTHONPATH=%PYTHONPATH%
 
 
 ECHO Checking for presence of Python in the system path variable.
-python --version >nul 2>&1
+python --version
```

Comment:
nit: make the equivalent change in the batch script

Reply:
(if there is an equivalent change to be made)

Reply:
Fixed